src/UI/Component/Deck/Deck.php

```diff
@@ -29,13 +29,75 @@ public function withCards($cards);
 	public function getCards();
 
 	/**
-	 * Set the cards size
+	 * Set the cards size to extra small:
+	 *  - 12 Cards on normal screens
+	 *  - 6 Cards on small screens
+	 *  - 1 Card on very small screens
+	 *
+	 * @param int Size of the card
 	 * @return Deck
 	 */
-	public function withCardsSize($size);
+	public function withExtraSmallCards();
 
 	/**
-	 * Get the cards size
+	 * Set the cards size to small:
+	 *  - 6 Cards on normal screens
+	 *  - 3 Cards on small screens
+	 *  - 1 Card on very small screens
+	 *
+	 * @param int Size of the card
+	 * @return Deck
+	 */
+	public function withSmallCards();
```

Comment:
Just an idea: how about renaming those methods to withSmallSize, withLargeSize, ... I was a bit confused by `->withCards(...)->withLargeCards()`. But I can live with both :-)

Reply:
... or even withLargeCardSize()...

Reply:
I appreciate Fabian's suggestion regarding the naming.